doc/source/examples/incompressible-flow/2d-transient-flow-around-cylinder/2d-transient-flow-around-cylinder.rst

```diff
@@ -0,0 +1,277 @@
+======================================
+2D Transient flow around a cylinder
+======================================
+
+This example corresponds to a transient flow around a fixed cylinder at a high Reynolds number.
+
+Features
+---------
+
+- Solver: ``gls_navier_stokes_2d`` (with Q2-Q1)
+- Transient problem
+- Usage of Gnuplot and Python scripts for the data post-processing
+
+Location of the example
+------------------------
+
+- Parameter file: ``/examples/incompressible_flow/2d_transient-flow_around_cylinder/cylinder.prm``
+- Mesh file: ``/examples/incompressible_flow/2d_transient-flow_around_cylinder/cylinder_structured.msh``
+- Geometry file: ``/examples/incompressible_flow/2d_transient-flow_around_cylinder/cylinder_structured.geo``
+
+Description of the case
+-----------------------
+We simulate the flow around a fixed cylinder with a constant upstream fluid velocity. We re-use the geometry and mesh presented in `2D Flow around a cylinder <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.html>`_, which were taken from Blais et al. [1]:
+
+.. image:: images/geometry_description.png
+    :alt: The geometry
+    :align: center
+    :name: geometry_description
+
+The flow field features a stable laminar boundary layer at the cylinder leading edge and a recirculation zone behind it formed by two unstable vortices of opposite signs. These vortices successively detach from the cylinder in a periodic manner (vortex shedding), leading to the generation of the von Kármán vortex street pattern in the wake. This vortex shedding causes a fluctuating pressure force acting on the cylinder, resulting in oscillations of the drag and lift coefficients in time. The frequency of vortex shedding is related to the Strouhal number:
+
+.. math::
+ S_t = \frac{D f_v}{U_\infty}
+
+where :math:`D` is the diameter of the cylinder, :math:`f_v` is the frequency of the shedding and :math:`U_\infty` is the upstream velocity.
+
+Parameter file
+--------------
+
+Simulation control
+~~~~~~~~~~~~~~~~~~
+This example uses a 2nd order backward differentiation (``method = bdf2``) for the time integration scheme. The simulation time is set to 200 seconds with the ``time end`` parameter and a time step of 0.05 seconds is used (``time step = 0.05``).
+
+.. code-block:: text
+
+    subsection simulation control
+        set method                = bdf2
+        set output name           = cylinder-output
+        set output frequency      = 1
+        set output path           = ./Re200/
```

Comment:
I create the folder (with a dummy file in it) in the example directory since there are many output files. I followed the structures of other examples to do so :)